src/_pytest/python_api.py

```diff
@@ -623,14 +627,6 @@ def raises(expected_exception, *args, **kwargs):
         >>> raises(ZeroDivisionError, f, x=0)
         <ExceptionInfo ...>
 
-    It is also possible to pass a string to be evaluated at runtime::
```

Comment:
The docstring above mentions "Legacy forms", but now only a single form is explained in the text. 😁